neutrons/reflectivity_ui
# Global configuration changed when selecting a run

# Short description of the changes:
Add `ConfigurationHandler` which connects signals from global configuration parameter UI widgets to callbacks updating the corresponding `Configuration` class variable.

# Check list for the pull request

# Check list for the reviewer

# Manual test for the reviewer
1. Load two runs (e.g. 42536 and 42537) and add them to the reduction table
2. Modify a value in the panel "Reflectivity Extraction (Global)"
3. Double click a different run in the column "Run No." in the reduction table
4. Verify that the config value changed in 2. is still the same

# References
[Defect 4758: [QUICKNXS] Global options updated when selecting a run](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20%28Change%20Management%29#action=com.ibm.team.workitem.viewWorkItem&id=4758)
